actionlist/app/components/action_item_body.js

```diff
@@ -1,10 +1,12 @@
+/* eslint no-underscore-dangle: ["error", { "allow": ["_links"] }] */
```

Comment:
This eslint directive should appear closer to where it is used (and re-enabled ASAP).